.claude/skills/api-platform-crud/reference/configuration-patterns.md

```
@@ -0,0 +1,238 @@
+## Operation Types
+
+| Operation     | HTTP Method | Purpose          | Input DTO      | Processor Required    |
+| ------------- | ----------- | ---------------- | -------------- | --------------------- |
+| GetCollection | GET         | List resources   | None           | No (default provider) |
+| Get           | GET         | Single resource  | None           | No (default provider) |
+| Post          | POST        | Create resource  | {Entity}Create | Yes                   |
+| Put           | PUT         | Full replacement | {Entity}Put    | Yes                   |
+| Patch         | PATCH       | Partial update   | {Entity}Patch  | Yes                   |
+| Delete        | DELETE      | Remove resource  | None           | No (default)          |
+
+**Example: Full CRUD Configuration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)